xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -240,7 +240,7 @@ extern "C"
       COLORTRC_MAX
     };
 
-    uint32_t m_flags;
+    uint32_t m_flags = STREAM_FLAGS::FLAG_NONE;
```

Comment:
Make for all them here on struct a:
```
INPUTSTREAM props = {{0}};
```
by https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp#L123

Reply:
Sounds good, and I changed it that way. However even though it compiles on my machine (gcc 9.2) the Jenkins build fails on some platforms now, see for example https://jenkins.kodi.tv/job/LINUX-RBPI/15289/console

Reply:
This is now fixed, all builds are green.